drivers/serial/Kconfig.pl011

```diff
@@ -21,4 +21,11 @@ config UART_PL011_SBSA
 	  among other things defines simplified UART interface
 	  which is subset of PL011 interface.
 
+config UART_PL011_AMBIQ_CLK_CR_REG
+	bool "Clock configuration via the CR register"
+	default y if SOC_APOLLO4P
+	help
+	  Enable configuring clock via the CR register. This feature
+	  is present in Ambiq Apollo4 SoCs.
```

Comment:
is this an Ambiq specific modification to the PL011 IP?

Reply:
Yes, this modification to PL011 core is specific to Ambiq SoCs.